overlord/snapstate/backend_test.go

```diff
@@ -51,8 +51,9 @@ type fakeOp struct {
 	sinfo   snap.SideInfo
 	stype   snap.Type
 
-	curSnaps []store.CurrentSnap
-	action   store.SnapAction
+	curSnaps    []store.CurrentSnap
+	action      store.SnapAction
+	refreshOpts *store.RefreshOptions
```

Comment:
mixed feeling about this, it feels a bit overkill,  maybe just keep a map[string]bool on  the fakeBackend and store seen salts there, and check they are set in a few crucial tests